example.gradle.properties

```diff
@@ -1,17 +0,0 @@
-#build time properties
```

Comment:
it was an example of gradle.properties file, I don't think that we should remove it